packages/base/Form/src/FieldsLayout/FieldsLayoutContext.tsx

```diff
@@ -22,6 +22,8 @@ const FieldsLayoutContext = createContext<FieldsLayoutContextValue>({
   labelWidth: DEFAULT_LABEL_WIDTH_SIZE,
 })
 
+export type Layout = 'horizontal' | 'vertical'
+
 export type FieldsLayoutContextProviderProps = {
   layout?: 'horizontal' | 'vertical'
```

Comment:
```suggestion
  layout?: Layout
```
I think this can now be replaced with the exported type